src/sentry/search/events/builder/base.py

```diff
@@ -1497,17 +1497,17 @@ def get_public_alias(self, function: CurriedFunction) -> str:
         """
         return self.function_alias_map[function.alias].field
 
-    def _get_dataset_name(self) -> str:
+    def _get_entity_name(self) -> str:
```

Comment:
This is a bit confusing since we're not actually returning entity keys. should we leave this as is and introduce a new method to the base class that returns the right entity based on the dataset you're querying?

Reply:
Added a `DATASET_TO_ENTITY_MAP` as discussed in slack!